### PR TITLE
Fix clinics submission import

### PR DESCRIPTION
**Pull request by KipSigei**
- Update client tool form ids
- Update health facility data
- Add script to delete all clinics and be able to re-import new / updated clinics into the dashboard
- Fix submissions import script
- Add push_locations endpoint

### CODE REVIEW

File: whoahqa/models/submission.py [hunk 1]
```diff
@@ -54,8 +55,11 @@ class Submission(Base):
 
     # tools to handler mapping
     HANDLER_TO_XFORMS_MAPPING = (
-        (ClinicReportHandler,
-         [tool for tool, label in constants.CLIENT_TOOLS]),
+        (ClinicReportHandler, [
+            tool
+            for tool, label in itertools.chain(
```
Comment (ivermac) [inline whoahqa/models/submission.py]: I think best practice when you don't use a variable like `label` in your case is to use an underscore.

**Comment by ivermac** [comment]
Other than the comment, it looks good.

File: whoahqa/models/submission.py (hunk 1 above)
Comment (KipSigei) [reply whoahqa/models/submission.py]: noted

**Comment by KipSigei** [comment]
@samwata please add https://whoahqa.ona.io/submissions/ webhook to all the forms once data cleaning is done

**Comment by samwata** [comment]
@KipSigei all the form and submissions are updated

**Pull request merged**